keyboards/handwired/stream_cheap/2x4/keymaps/via/rules.mk

```
@@ -0,0 +1,10 @@
+VIA_ENABLE = yes
+# LTO: link time optimization makes the build take slightly longer
+# but makes the resulting .hex file smaller, which allows you to
+# fit more features into smaller MCUs:
+LTO_ENABLE = yes
+# Support for these features make the hex file larger, but we want 'em:
+MOUSEKEY_ENABLE = yes     # Allow mapping of mouse control keys
+EXTRAKEY_ENABLE = yes     # Allow audio & system control keys
+COMMAND_ENABLE = yes      # Commands for debug and configuration
+NKRO_ENABLE = yes         # Support USB N-key roll over.
```

```suggestion
VIA_ENABLE = yes
LTO_ENABLE = yes
```
Command is not all that useful on a board of this size. The rest are already enabled at keyboard level.

---

Addressed in 3bb3a04